roles/ci_gen_kustomize_values/templates/ovs-dpdk-sriov/edpm-values/values.yaml.j2

```diff
@@ -0,0 +1,74 @@
+{% set instances_names = []                                                            %}
+{% set _original_nodeset = (original_content.data | default({})).nodeset | default({}) %}
+{% set _original_nodes = _original_nodeset.nodes | default({})                         %}
+{% set _vm_type = (_original_nodes.keys() | first).split('-')[1]                       %}
+{% for _inst in cifmw_networking_env_definition.instances.keys()                       %}
+{%   if _inst.startswith(_vm_type)                                                     %}
+{%     set _ = instances_names.append(_inst)                                           %}
+{%   endif                                                                             %}
+{% endfor                                                                              %}
+data:
+  root_password: {{ cifmw_ci_gen_kustomize_values_root_password | b64encode }}
+  preProvisioned: {{ cifmw_ci_gen_kustomize_values_pre_provisioned | b64encode }}
+  baremetalSetTemplate:
+    ctlplaneInterface: {{ cifmw_networking_env_definition.instances[instance].networks.ctlplane.interface_name }}
```

Comment:
Are you sure this is the value you want here? iiuc correctly this value is used in out OSP BMO to render [this](https://github.com/openstack-k8s-operators/openstack-baremetal-operator/blob/77086641d3fd316776dc9f771bd20b0f593cba29/templates/openstackbaremetalset/cloudinit/networkdata#L6)

Reply:
@pablintino 
We have covered the customization in architecture repo. This role generates VA2 architecture edpm values.yaml  format only right? not OSP BMO CR